### PR TITLE
Update contributing guide to change OpenAPI doc on a release

### PR DESCRIPTION
**Pull request by mattmcneeney**

### CODE REVIEW

**Comment by angarg12** [comment]
LGTM

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -175,6 +175,8 @@ any PMC member:
 1. Create a new commit updating the [spec.md](spec.md) and [profile](profile.md)
    headers to include the version of the release
    (`Open Service Broker API (v$major.$minor)`).
+1. Create a new commit updating the [oepnapi.yaml](openapi.yaml) version to be
```
Comment (duglin) [inline CONTRIBUTING.md]: Not sure why you didn't just include the yml in the previous list of files to update, but ok...

**Comment by duglin** [comment]
LGTM

File: CONTRIBUTING.md (hunk 1 above)
Comment (mattmcneeney) [reply CONTRIBUTING.md]: Yeah. I don't know why either. Updating!

Comment (mattmcneeney) [reply CONTRIBUTING.md]: (also realised I spelt `openapi.yaml` wrong!

**Comment by mattmcneeney** [comment]
Rebased @angarg12 @duglin

**Comment by angarg12** [comment]
LGTM

**Comment by duglin** [comment]
LGTM

**Comment by duglin** [comment]
2 more reviews needed

**Comment by duglin** [comment]
2 more needed - @pmorie @vaikas-google ?

**Comment by duglin** [comment]
@avade @fmui

**Comment by fmui** [comment]
LGTM

**Comment by zrob** [comment]
LGTM

**Pull request merged**